input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv

```diff
@@ -828,7 +828,7 @@ D = Donation
 E = Entombment
 R = Removal from state
 O = Other
-U = Unknown",,VRDR,DecedentDispositionMethod,value,codeable,[FetalRemainsDispositionMethodVS],,y,,,,DecedentDispositionMethod,
```

Comment:
I don't think this is right

Reply:
This is VRDR DecedentDispositionMethod which uses MethodOfDispositionVS